os/include/tinyara/wifi/rtk/wifi_structures.h

```diff
@@ -134,17 +134,39 @@ typedef struct rtw_scan_handler_result {
   * @brief  The structure is used to describe the cfg parameters used for csi report,
   * @note  The mac_addr if not specified, the default value must be 0.
   */
-#ifdef CONFIG_ARCH_CHIP_RTL8720E
+#if CONFIG_ARCH_CHIP_RTL8720E || CONFIG_ARCH_CHIP_RTL8730E

```

Comment:
2nd commit for app includes changes on the os. Please move this into the 1st commit.

Reply:
Changed accordingly